packages/blockprotocol/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "allowJs": true,
     "isolatedModules": true,
-    "jsx": "preserve",
+    "jsx": "react-jsx",
```

Comment:
As above RE typescript/JSX.

Reply:
Replied in https://github.com/blockprotocol/blockprotocol/pull/441#r925489980